option optimize_for = LITE_RUNTIME;
import "U64Type.proto";
message LegionOperateAnswer
{
 required uint32 result = 1;
 required U64Type playerid = 2;
}
